src/Workspaces/VisualBasic/Portable/CodeCleanup/Providers/ReduceTokensCodeCleanupProvider.vb

```diff
@@ -71,6 +71,28 @@ Namespace Microsoft.CodeAnalysis.CodeCleanup.Providers
                         If Not CaseInsensitiveComparison.Equals(valueText, idText) Then
                             Return newNode.ReplaceToken(literal, CreateDecimalLiteralToken(literal, valueText, value))
                         End If
+                    Case SyntaxKind.IntegerLiteralToken
+                        ' Get the literal identifier text which needs to be pretty listed.
+                        Dim idText = literal.GetIdentifierText()
+                        Dim value As Long = CType(literal.Value, Long)
+
+                        If value = 0 Then
+                            ' Overflow/underflow case or zero literal, skip pretty listing.
+                            Return newNode
+                        End If
+
+                        Dim base = literal.GetBase()
+
+                        If Not base.HasValue Then
+                            Return newNode
+                        End If
+
+                        'fetch the string representation of this value in the correct base.
+                        Dim valueText As String = GetIntegerLiteralValueString(value, base.Value) + GetTypeCharString(literal.GetTypeCharacter())
```

Comment:
I based this on the clean-ups for DecimalLiteralToken and FloatingLiteralToken which also use + to generate the value string. I'm assuming these should also be changed over to use the concatentation operator.